core/controllers/question_editor.py

```diff
@@ -115,19 +115,13 @@ class QuestionSkillLinkHandler(base.BaseHandler):
     """A handler for linking and unlinking questions to or from a skill."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, question_id, skill_id):
```

Comment:
This function is never used, question skill link will always be created during creation of the question itself.